### ISSUE TITLE
Can you add: bitcore-lib / insight

### ISSUE THREAD

**Issue by HebeBlock** [abcmint/abcmint#22]
I am adding abc to our wallet, but found that there are no 2 libraries。
If you can add them, we will be able to add abc to my wallet。

**Comment by gisfjb** [comment]
> I am adding abc to our wallet, but found that there are no 2 libraries。
> If you can add them, we will be able to add abc to my wallet。

hi, what's the name of the "2 libraries", and what are them use for?  any reference? @HebeBlock

**Comment by gisfjb** [comment]
Block explorer (insight)
Block, transaction, and wallet utilities (bitcore-lib)
yes, i think it should be added.

**Comment by windimhaar** [comment]
it should be tools developed by js.  you want something like this
https://github.com/bitpay ?
that would be lots of work. @gisfjb @HebeBlock

**Comment by HebeBlock** [comment]
@windimhaar Yes, such a thing is needed so that the mobile wallet can be developed quickly

**Comment by liamstojanovic** [comment]
> @windimhaar Yes, such a thing is needed so that the mobile wallet can be developed quickly

I was looking at the following repo: https://github.com/bitpay/bitcore

Would a proper config for Bitcore/ABCmint enable the functionality you require?
